pkgs/tools/package-management/lix/default.nix

```
@@ -57,6 +60,14 @@ lib.makeExtensible (self: ({
     }
   );
 
-  latest = self.lix_2_90;
-  stable = self.lix_2_90;
-}))
+  lix_2_91 = (
+    common {
+      version = "2.91.0";
+      hash = "sha256-Rosl9iA9MybF5Bud4BTAQ9adbY81aGmPfV8dDBGl34s=";
+      docCargoHash = "sha256-KOn1fXF7k7c/0e5ZCNZwt3YZmjL1oi5A2mhwxQWKaUo=";
```

This is the merged patch from the gerrit ticket above
```suggestion
      docCargoHash = "sha256-KOn1fXF7k7c/0e5ZCNZwt3YZmjL1oi5A2mhwxQWKaUo=";
      
      patches = [
        # TODO: Remove next release
        # Fixes cross compilation targets
        # https://gerrit.lix.systems/c/lix/+/1762
        (fetchpatch {
          url = "https://git.lix.systems/lix-project/lix/commit/41a0b08e646b54b27ae49fb4c23e0c45c60369ca.patch";
          hash = "sha256-TZauU4RIsn07xv9vZ33amrDvCLMbrtcHs1ozOTLgu98=";
        })
      ];
```